app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,27 @@
+class BookingsController < ApplicationController
+  def new
+    @property = Property.find(params[:property_id])
+    @booking = @property.bookings.new
+  end
+
+  def create
+    @property = Property.find(params[:property_id])
+    @booking = @property.bookings.new(booking_params)
```

Comment:
the models booking and property are already connected through a foreign key, therefore adding the property_id on the reviews table/model is redundunt. consider removing it, if you keep it, no big deal :)